src/python/pants/backend/native/tasks/native_external_library_fetch.py

```diff
@@ -179,9 +179,7 @@ def ensure_conan_remote_configuration(self, conan_binary):
     # and replace it with Pants-controlled remotes.
     remove_conan_center_remote_cmdline = self._remove_conan_center_remote_cmdline(conan_binary)
     try:
-      # Slice the command line because subprocess errors when the first element in the
-      # list of command strings is the setting of an environment variable.
-      stdout = subprocess.check_output(remove_conan_center_remote_cmdline.split()[1:])
```

Comment:
This is fixed not _only_ if pex path is not specified, correct? Based on the changes to `wrapped_pex` below, it seems this would still be needed if PEX_PATH were set.

Reply:
This code no longer uses WrappedPEX.